memgpt/server/rest_api/server.py

```diff
@@ -0,0 +1,112 @@
+import asyncio
+import json
+from typing import Union
+
+from fastapi import FastAPI, HTTPException
+from fastapi.responses import StreamingResponse
+from pydantic import BaseModel
```

Comment:
@sarahwooders should we be adding `pydantic` to `.toml` if we're direct importing? I'm assuming the dep is handled via a nested dep inside of `fastapi`.

Reply:
https://github.com/cpacker/MemGPT/blob/843c69d5d47ddd8fd11becf3ac6d509173651cc0/poetry.lock#L738-L753

Reply:
added as a dep to toml